src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3157,4 +3157,8 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Deleting_0_requires_restarting_the_application_because_is_not_supported_by_the_runtime" xml:space="preserve">
     <value>Deleting {0} requires restarting the application because is not supported by the runtime.</value>
   </data>
+  <data name="Extract_base_record" xml:space="preserve">
+    <value>Extract base record...</value>
+    <comment>"record" is a language construct and should be left the same</comment>
```

Comment:
Consider using `{Locked="record"}` similar to this:

https://github.com/dotnet/roslyn/blob/2552d53c0a044cc5301c195cde161739e89dfc28/src/Features/CSharp/Portable/CSharpFeaturesResources.resx#L131